app/assets/stylesheets/styles.scss

```diff
@@ -25,9 +118,9 @@ html {
         margin: 0px;
         flex-flow: column nowrap;
         align-items: stretch;
-        background: #f7feff;
+        background: radial-gradient(ellipse at bottom, #1B2735 0%, #090A0F 100%);
```

Comment:
we may need to add vendor specific prefixes to radial gradient(-moz and -webkit) and also specifiy a background color if it fails to render. IE support is as usual, lagging behind, but in this case I think the alternative is very simple (adding bgcolor) and we should do it.